docs/how_to/noisy_simulators.rst

```diff
@@ -0,0 +1,430 @@
+``Sampler`` and ``Estimator`` to study the effects of errors which occur
+on real devices.
+
+Set up your local development environment
+-----------------------------------------
+
+This tutorial requires a Qiskit Runtime service instance to be setup. If
+you haven’t done so already, please follow `these
+steps <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/getting_started.html>`__
+to set one up.
+
+.. code-block:: python
+
+    # load necessary Runtime libraries
+    from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Estimator, Session, Options
+
+    service = QiskitRuntimeService(channel="ibm_quantum")
+
+Preparing the environment
+-------------------------
+
+To demonstrate the routine, we shall proceed with running an example
+routine. One of the major benefits of using primitives is simplification
+of binding multiple parameters in parameterized circuits. To check this,
+here is an example circuit with a controlled
+`P-gate <https://qiskit.org/documentation/stubs/qiskit.circuit.library.PhaseGate.html>`__
+as implemented right below. Here, we parametrise the ``P-gate`` with a
+rotation parameter ``theta``. To learn how to create circuits and bind
+parameters to them by using Qiskit, see the `Circuit
+Basics <https://qiskit.org/documentation/tutorials/circuits/01_circuit_basics.html>`__
+and `Advanced
+Circuits <https://qiskit.org/documentation/tutorials/circuits_advanced/01_advanced_circuits.html#Parameterized-circuits>`__
+in Qiskit documentation.
+
+.. code-block:: python
+
+    from qiskit.circuit import Parameter
+    from qiskit import QuantumCircuit
+
+    theta = Parameter('theta')
+
+    qc = QuantumCircuit(2,1)
+    qc.x(1)
+    qc.h(0)
+    qc.cp(theta,0,1)
+    qc.h(0)
+    qc.measure(0,0)
+
+    qc.draw('mpl')
+
+
+
+
+.. image:: /docs/images/noisy-sim-circuit.png
+
+
+
+The circuit shown above is parameterized with the eigenvalue being
+kicked back into qubit 0 to be measured. The amount of kickback will be
+determined by the parameter theta. Now in the cell below, we shall
+define our parameters for our above circuit as a list. The parameters
+here will be from :math:`0` to :math:`2\pi` divided over 50 evenly
+spaced points.
+
+.. code-block:: python
+
+    import numpy as np
+
+    phases = np.linspace(0, 2*np.pi, 50)
+
+    # phases need to be expressed as a list of lists in order to work
+    individual_phases = [[phase] for phase in phases]
+
+Running on the ideal simulator
+------------------------------
+
+Set the backend and options to use
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First we shall demonstrate a run using an ideal case without any
+``noise_model``, ``optimization_level`` or ``resilience_level`` for both
+Sampler and Estimator. We shall proceed to setup the options as shown
+below:
+
+.. code-block:: python
+
+    backend = "ibmq_qasm_simulator" # use the simulator
+
+.. code-block:: python
+
+    options = Options()
+    options.simulator = {"seed_simulator": 42}
```

Comment:
Not really a big deal, but you can also do `options.simulator.seed_simulator = 42` instead of using a dictionary.

Reply:
I've changed it as you suggested. Thank you! :)